Error zauth_module

When executing 
```
# build nginz
cd services/nginz && make
```
the poppy produces an error

```
../third_party/nginx-zauth-module/zauth_module.c:6:19: fatal error: zauth.h: No such file or directory
compilation terminated.
objs/Makefile:1253: recipe for target 'objs/addon/nginx-zauth-module/zauth_module.o' failed
make[2]: *** [objs/addon/nginx-zauth-module/zauth_module.o] Error 1
make[2]: Leaving directory '/root/wire-server/services/nginz/src'
Makefile:8: recipe for target 'build' failed
make[1]: *** [build] Error 2
make[1]: Leaving directory '/root/wire-server/services/nginz/src'
Makefile:55: recipe for target 'src/objs/nginx' failed
make: *** [src/objs/nginx] Error 2

```

--- a/README.md
+++ b/README.md
@@ -76,9 +76,9 @@ Once all dependencies are set up, the following should succeed:
 make
 # build one haskell service, e.g. brig:
 cd services/brig && make
-# build nginz
-cd services/nginz && make
 ```
+
+For building nginz, see [services/nginz/README.md](services/nginz/README.md)
 
 #### 2. Use docker
 

--- a/services/nginz/README.md
+++ b/services/nginz/README.md
@@ -1,1 +1,18 @@
 # NGINX build with extra modules
+
+## Compile natively
+
+To build nginz natively, ensure to have the usual C compiler toolchains installed, along with the following dependencies:
+
+* openssl
+* libossp-uuid
+* [libzauth](../../libs/libzauth) 
+    * depends on the rust compiler, libsodium, [makedeb](../../tools/makedeb)
+
+If you're on alpine, see the [Dockerfile](Dockerfile) for the precise dependency names. If you're on another platform, their names might differ slightly.
+
+Once you have all necessary dependencies, `make` in this directory should work.
+
+## Compile with docker
+
+`make docker`

